packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -184,10 +184,12 @@ function getAndUpdateNormalizedValues(field: RhfFormFields[number], t: TFunction
     Object.entries(field.variantsConfig.variants).forEach(([variantName, variant]) => {
       variant.fields.forEach((variantField) => {
         const fieldTypeVariantsConfig = fieldTypesConfigMap[field.type]?.variantsConfig;
-        const defaultVariantFieldLabel =
-          fieldTypeVariantsConfig?.variants?.[variantName]?.fieldsMap[variantField.name]?.defaultLabel;
-
-        variantField.label = variantField.label || t(defaultVariantFieldLabel || "");
+        const fieldTypeVariantConfig = fieldTypeVariantsConfig?.variants[variantName];
+        variantField.label = getVariantFieldLabel({
```

Comment:
@ZainGulbaz Reused `getVariantFieldLabel` here to make the implementation consistent with the app that you implemented. Earlier, if just spaces are entered, then booker was showing just the spaces and not the fallback, now with your fix it would fallback to default.